### DISCUSSION TITLE
keybind to open the glance popover in new tab instead of clicking

### DISCUSSION THREAD

**Discussion by KorigamiK** [zen-browser/desktop#3428] [Ideas]
I want to expand the glance popover into a new tab, but I can't do that without using my mouse. 
A keybind just for this will be ideal!

**Comment by drornir** [comment]
This thread was opened in the "Q&A" section instead in "Ideas" so I'm linking this here

https://github.com/zen-browser/desktop/discussions/5079

Just for reference: In Arc, you can press cmd+o to open the glance window as a tab.

**Comment by ArshiAAkhavan** [comment]
Silly Idea but i use Vimium extension on Zen, and it supports opening links via key strocks (F + link_refrence_char) so if there was a local Zen link behind this button, i.e. zen://open-new-tab which opens the current preview in new tab if pressed, theoretically we could have the short cut with vimium extension.

**Reply by KorigamiK** [reply]
This is not the same. You can open it using the glance popover before when you use vimium to open in background or new tab.

**Reply by ArshiAAkhavan** [reply]
I didn't quite get it, can you elaborate further? are you suggesting I can setup a keyshort cut with vimium for expansion button?

my setup up right now is that when I click the F<link_char> in caps mode, vimium by default tries to open it in a new window, but since the glance popover option is enabled on zen, instead of a new tab, it shows the pop over view, from then, in order to expand it into a new tab, I just have to manually click on the icon with my mouse. what configuration should I change?

**Reply by KorigamiK** [reply]
No I'm saying after you open the glance window, we can have a shortcut that's not related to vimium to open the glance window in a new tab
